cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -439,6 +452,121 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           resultTable()
         }
       }
+
```

Comment:
Maybe we should add to the operations manual examples for DBMS and DATABASE privileges as well, I think the healthcare example is just GRAPH privileges (plus one admin)?

Might be a different card and not as part of this but just a thought I had that we maybe should consider it

Reply:
I believe that the docs team should be responsible for the operations manual so it is presented in a nice way. But we should probably sync about it with them